docs/source/_json/content_patch_representation.req

```diff
@@ -0,0 +1,8 @@
+PATCH /plone/folder/my-document HTTP/1.1
+Accept: application/json
+Authorization: Basic YWRtaW46c2VjcmV0
+Content-Type: application/json
```

Comment:
The `Prefer` header is missing in the example. You need to add it to the list of request header keys https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/tests/test_documentation.py#L56

Reply:
I missed that, thanks for the review.